backend/tests/Feature/Web/TutorialModuleTest.php

```
@@ -0,0 +1,66 @@
+        // Step 5: the dashboard KPI widget is registered.
+        $widgets = app(WidgetRegistry::class);
+        $this->assertTrue($widgets->hasWidgets('kpi'));
+        $this->assertSame('Order Pinger', $widgets->getWidgets('kpi')[0]['title']);
+
+        // Step 6: the module's own page route exists.
+        app('router')->getRoutes()->refreshNameLookups();
+        $this->assertTrue(app('router')->has('order-pinger.status'));
+
+        // End to end: real saves fire the events and run the module's handlers.
+        // A bug in Hooks.php (bad method/reference) would throw here.
+        $wo = WorkOrder::factory()->create();       // → WorkOrderCreated + ResourceChanged
```

PHPStan (2.2.6)

[error] Call to an undefined static method App\Models\WorkOrder::factory().


(staticMethod.notFound)